src/executables/util_service.c

```diff
@@ -3205,6 +3211,35 @@ ha_argv_to_args (char *args, int size, const char **argv, HB_PROC_TYPE type)
   return status;
 }
 
+static int
+us_hb_process_rkcheck (HA_CONF * ha_conf, const char *db_name)
+{
+  int status = NO_ERROR;
+  char **dbs;
+  int i;
+
+  dbs = ha_conf->db_names;
+
+  for (i = 0; dbs[i] != NULL; i++)
+    {
+      db_name = dbs[i];
+      const char *lw_argv[] = { UTIL_ADMIN_NAME, UTIL_RKCHECK,
```

Comment:
NIT:
lw_는 log writer(== copylogdb)의 prefix 입니다. 
```suggestion
      const char *rkcheck_args[] = { UTIL_ADMIN_NAME, UTIL_RKCHECK,
```

Reply:
@hornetmj 반영했습니다.